test/runtests.jl

```diff
@@ -85,6 +85,8 @@ ex = x^2 + y^2
 @test_approx_eq lambdify(sin(Basic(1))) sin(1)
 ex = sin(x)
 @test_approx_eq lambdify(ex)(1) sin(1)
+ex = exp(PI/2*x)
+@test_approx_eq lambdify(ex)(1) exp(pi/1)
```

Comment:
Sorry, thanks. Fixed now.

On Fri, Apr 8, 2016 at 1:38 PM, Isuru Fernando <notifications@github.com>
wrote:

> In test/runtests.jl
> <https://github.com/symengine/SymEngine.jl/pull/21#discussion_r59062020>:
>
> > @@ -85,6 +85,8 @@ ex = x^2 + y^2
> >  @test_approx_eq lambdify(sin(Basic(1))) sin(1)
> >  ex = sin(x)
> >  @test_approx_eq lambdify(ex)(1) sin(1)
> > +ex = exp(PI/2*x)
> > +@test_approx_eq lambdify(ex)(1) exp(pi/1)
>
> This should be exp(pi/2)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symengine/SymEngine.jl/pull/21/files/6abfc71870547a28375b7487f1aa77f78f6634a6#r59062020>
>



-- 
John Verzani
Chair, Department of Mathematics
College of Staten Island, CUNY
verzani@math.csi.cuny.edu